Library/Homebrew/resource_auditor.rb

```diff
@@ -124,6 +124,21 @@ def audit_urls
       end
     end
 
+    def audit_head_branch
+      return if !@online || !@strict || spec_name != :head || !Utils::Git.remote_exists?(url)
+
+      branch = Utils.popen_read("git", "ls-remote", "--symref", url, "HEAD")
+                    .match(%r{ref: refs/heads/(.*?)\s+HEAD})[1]
+
+      return if branch == "master" && specs[:branch].blank? || branch == specs[:branch]
+
+      if branch == "master"
+        problem "Remove `branch: \"#{specs[:branch]}\"`"
```

Comment:
I'm tempted to say we want the branch to be always set so that it'll make it easier for us to change the default in future. Thoughts?

Reply:
Big 👍 to this